requirements.txt

```diff
@@ -1,5 +1,6 @@
 discord.py
 discord.py[voice]
+requests
```

Comment:
is this needed?

Reply:
Yes

Reply:
it was used without being added in requirements.